app/scripts/controllers/client/CreateClientController.js

```diff
@@ -51,8 +62,62 @@
                         scope.formData.staffId = data.staffId;
                     }
                 }
+
+                scope.addressTypes=data.address.addressTypeIdOptions;
+                scope.countryOptions=data.address.countryIdOptions;
+                scope.stateOptions=data.address.stateProvinceIdOptions;
+                scope.enableAddress=data.isAddressEnabled;
+
+                if(scope.enableAddress===true)
+                {
+                    resourceFactory.addressFieldConfiguration.get({entity:entityname},function(data){
+                        scope.addresstypId=data[0].is_enabled;
```

Comment:
Usually, we should not have hard coded indexes like this.  Problem with this implementation is in future somebody adds a new type in between these indexes, that person need to comeback and look into these existing indexes also.